### PR TITLE
fsdb path config handling. fixes #38

### LINKED ISSUE #38: fsdb path
Now fsdb path is hardcoded as ```/tmp/fsdbRoot```. This sucks.

Following other settings style we will allow to pass this specific config through variable ```WEBANT_FSDB_PATH```, but some questions need answers:
What should append if user do not pass fsdb path?
Which path should we use as the default one?

### PR DESCRIPTION
**Pull request by ael-code**
fixes #38

### CODE REVIEW

File: webant/webant.py [hunk 1]
```diff
@@ -48,8 +49,15 @@ def create_app(configfile=None):
 
     presetManager = PresetManager(app.config['PRESET_PATHS'])
 
-    #TODO change fsdb path
-    fsdb = Fsdb("/tmp/fsdbRoot")
+    if not app.config['FSDB_PATH']:
+        print "not fsdbpath {}".format(app.config['DEBUG'])
```
Comment (boyska) [inline webant/webant.py]: this should be removed!

**Comment by boyska** [comment]
There is just a very minor ugliness (a `print` statement). Apart from that, it is perfect.

File: webant/webant.py (hunk 1 above)
Comment (ael-code) [inline webant/webant.py]: hahaha 
sorry man

**Comment by ael-code** [comment]
> There is just a very minor ugliness (a print statement). Apart from that, it is perfect.

fixed

**Comment by boyska** [comment]
EDDAJE!

**Pull request merged**